headers/diffs.diff

```diff
@@ -809,23 +1285,23 @@
  } Il2CppClass;
  
  typedef struct Il2CppClass_0 {
-@@ -831,7 +845,6 @@ typedef struct Il2CppClass_1 {
+@@ -831,7 +852,6 @@ typedef struct Il2CppClass_1 {
      uint32_t cctor_started;
      uint32_t cctor_finished;
  #ifdef IS_32BIT
 -    uint32_t cctor_thread__padding;
```

Comment:
This change is slightly alarming - the padding member exists to force `cctor_thread` to an alignment of 8 on 32-bit platforms. The roundabout implementation here was due to the class splitting (the split class `Il2CppClass_1` is not 8-byte-aligned within the full `Il2CppClass` on some versions in 32-bit mode). I don't see any other changes that would cause this member to move. Could you ensure the offsets and such are still right by running `offsettest.sh`?

Reply:
after `apt-get install gcc-multilib` and `apt-get install g++-multilib` so I could compile the 32-bit builds:

![image](https://user-images.githubusercontent.com/4500319/95404943-b1b08c00-0916-11eb-8020-a8519b6526c0.png)